### ISSUE TITLE
Intermittent error

### ISSUE THREAD

**Issue by robmikh** [robmikh/dwmdumper#1]
Sometimes processdumper gives the following error: 

```
0xD0000004 - The specified information record length does not match the length required for the specified information class.
```

**Comment by robmikh** [comment]
Looks like the buffer is too small here: https://github.com/robmikh/dwmdumper/blob/e94bd3742da69cbd98ed2b04910df9f6408f8d06/processdumper/src/process.rs#L26-L42

**Comment by robmikh** [comment]
Fixed with 356aa96726f877b822ffd26a5601c644c53eadbc
